godot-codegen/src/class_generator.rs

```diff
@@ -297,13 +297,17 @@ fn make_builtin_class(
             sys_ptr: sys::GDExtensionTypePtr,
         }
         impl<'a> #inner_class<'a> {
-            pub fn from_outer(outer: &#outer_class) -> Self {
+            pub fn from_sys_ptr(sys_ptr: sys::GDExtensionTypePtr) -> Self {
                 Self {
                     _outer_lifetime: std::marker::PhantomData,
-                    sys_ptr: outer.sys(),
+                    sys_ptr,
                 }
             }
 
+            pub fn from_outer(outer: &#outer_class) -> Self {
+                Self::from_sys_ptr(outer.sys())
+            }
+
```

Comment:
Why this change? The new method `from_sys_ptr` breaks the link from the lifetime in `&'a outer_class` to `#inner_class<'a>`, and returns [an unbounded lifetime](https://doc.rust-lang.org/nomicon/unbounded-lifetimes.html).

It doesn't seem like you really need it though?

Reply:
`from_outer` only exists on `Array`, not on `TypedArray<T>`. But I can handle that with another transmute on the caller side.

Reply:
Ah, I see. Both unbounded lifetime and transmute is not ideal, but I'd prefer the latter.
Maybe add a `TODO`, then at some point in the future, I can revisit this 😉